packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -66,9 +68,14 @@ async function buildSwingset(
     config = loadBasedir(vatconfig);
   }
 
-  const mbs = buildMailboxStateMap(mailboxStorage);
+  const mailboxStateMap = buildMailboxStateMap(mailboxStorage);
+  const activeMailboxStateMap = {
```

Comment:
If `mailboxStateMap` is used only to create `activeMailboxStateMap`, and otherwise should not be used, I would prefer:
```suggestion
  const rawMailboxStateMap = buildMailboxStateMap(mailboxStorage);
  const mailboxStateMap = {
```